python/tidy/servo_tidy/tidy.py

```diff
@@ -730,6 +734,19 @@ def parse_config(content):
             config[pref] = user_configs[pref]
 
 
+def check_directory_files(directories_for_files):
+    for directory, file_extensions in directories_for_files.items():
+        files = get_file_list(directory, only_changed_files=False, exclude_dirs=[])
+        for filename in files:
+            expected = [True for possible in file_extensions if filename.endswith(possible)]
+            if not expected:
```

Comment:
@Ms2ger @Wafflespeanut thank you both for your review, I have implemented the suggested changes, please take another look...